src/api/app/helpers/webui/user_or_groups_roles_helper.rb

```diff
@@ -1,9 +1,13 @@
 module Webui::UserOrGroupsRolesHelper
   def display_name(object)
     if object.is_a?(User) && object.realname.present?
-      "#{object.name} (#{object.login})"
+      content_tag :span do
+        concat "#{object.name} ("
+        concat content_tag(:i, object.logini.to_s)
```

Comment:
Typo in `object.login`.

`.to_s` isn't needed since `login` is already a string.